.github/workflows/build-image.yml

```diff
@@ -121,19 +121,47 @@ jobs:
     if: ${{ needs.detect-changes.outputs.cuda == 'true' }}
     strategy:
       fail-fast: false
+      # multi-OS build matrix: creates 4 images (2 platforms × 2 OS variants)
+      # - RHEL (ubi9): default/production image (no suffix in final tags)
+      # - Ubuntu (24.04): alternative base for compatibility testing (-ubuntu suffix)
```

Comment:
Will there be any interaction here with upstream vLLM? Don't they specifically build against Ubuntu 20.04 and 22.04?

Reply:
we implemented this, resolving